ClearComputeNode.ps1

```diff
@@ -0,0 +1,156 @@
+Param (
+    [Parameter(Mandatory = $false)] [String] $AdapterName = "Ethernet1",
+    [Parameter(Mandatory = $false)] [String] $ForwardingExtensionName = "vRouter forwarding extension",
+    [Parameter(Mandatory = $false)] [String] $VMSwitchName = "Layered Ethernet1",
+    [Parameter(Mandatory = $false)] [String] $ConfigAndLogDir = "C:\ProgramData\Contrail",
+    [Parameter(Mandatory = $false)] [String] $InstallationDir = "C:\Program Files\Juniper Networks"
+)
+
+function Stop-ProcessIfExists {
+    Param ([Parameter(Mandatory = $true)] [string] $ProcessName)
+
+    $Proc = Get-Process $ProcessName -ErrorAction SilentlyContinue
+    if ($Proc) {
+        $Proc | Stop-Process -Force -PassThru | Wait-Process -ErrorAction SilentlyContinue
+    }
+}
+
+function Remove-HNSNetworks {
+    Write-Host "Cleaning HNS state..."
+    Get-NetNat | Remove-NetNat -Confirm:$false
+    Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
+    Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
+}
+
+function Stop-NodeMgr {
+    Write-Host "Stopping Node Manager and removing service..."
+    $NodeMgrServiceName = "contrail-vrouter-nodemgr"
+    $Service = Get-Service $NodeMgrServiceName -ErrorAction SilentlyContinue
+    if ($Service -ne $null) {
+        Stop-Service $NodeMgrServiceName -ErrorAction SilentlyContinue
+        sc.exe delete $NodeMgrServiceName
```

Comment:
function name `Stop-NodeMgr` suggests it will stop it, not delete/uninstall it - this may lead to subtle bugs when someone tries to use `Stop-NodeMgr` function somewhere. Likewise for other functions like this one